src/strategies/latin_hypercube.jl

```diff
@@ -33,12 +33,21 @@ mutable struct LatinHypercube <: TuningStrategy
     interSampleWeight::Number
     ae_power::Number
     periodic_ae::Bool
+    rng::Random.AbstractRNG
 end
```

Comment:
I think this needs to be `Union{Integer, Random.AbstractRNG`, no?

Reply:
I was under the impression that it should behave the same as RandomSearch, no? So then should I put `Union{Integer,Random.AbstractRNG}` in random search as well?

Reply:
Oops. No, you are right! The constructor changes the type. I was getting confused with the model API, where there is not allowed.